src/components/ContextualMenu/ContextualMenu.less

```diff
@@ -8,19 +8,20 @@
 
 .ms-ContextualMenu {
   .ms-font-m;
-  .ms-u-normalize;
+  .resetMargins;
+  .resetPadding;
```

Comment:
`.ms-u-normalize` contains `.resetBoxShadow` which is not desired, because menu has `.drop-shadow`. But now I see that I missed `.ms-u-borderBox` to copy from `.ms-u-normalize`. So `.ms-ContextualMenu` should contains all styles from `.ms-u-normalize` except `.resetBoxShadow`. Sorry for missing `.ms-u-borderBox`